signbank/editorial_queue.py

```diff
@@ -0,0 +1,29 @@
+from .dictionary.models import Gloss
+from django.shortcuts import render
+from django_comments import get_model as django_comments_get_model
+
+
+def get_queue_items(request):
+    """
+        As per the first comment of N2-92, get the glosses which are assigned to you
+        """
+    user = request.user
+    print('URL **** ', request.path)
```

Comment:
Is this print() statement meant to be here?

Reply:
No, it's not. Removed it, thanks for highlighting :slightly_smiling_face: